upup/models/nodeup/_kubernetes_master/etcd/_not_protokube/etc/kubernetes/manifests/etcd-events.manifest

```diff
@@ -3,7 +3,10 @@
 "kind": "Pod",
 "metadata": {
   "name":"etcd-server-events",
-  "namespace": "kube-system"
+  "namespace": "kube-system",
+  "labels": {
+    "k8s-app" : "etcd-server-events"
```

Comment:
These files are probably dead and should be removed, but thanks for updating them anyway :-)